### DISCUSSION TITLE
Add support for multiple product pictures

### DISCUSSION THREAD

**Discussion by Janfred** [pretix/pretix#1956] [Ideas]
In our use case we want to have multiple pictures for a product. (In this special case a picture of a shirt and a size table). We were able to include a new picture in the same lightbox with some hacky things (like editing the allowed html tags) but I think this is a feature for upstream.

**Comment by raphaelm** [comment]
Hmm, since the product description allows markdown, I think it would be fine to link the sizing table there, wouldn't it?

**Comment by Janfred** [comment]
Not really, this link would open in the same tab. Also `img`-Tags are not allowed, so it's not allowed to include the image itself there. The second problem is, that pretix doesn't allow uploading additional assets, so the picture has to be hosted somewhere else.

A typical use case would be to show the front and back view of a conference shirt.

**Comment by rixx** [comment]
I'd see this as a low-prio feature request, as multiple images per product can make sense, but at the same time, the issue can be avoided until then in several ways (link the other information, or in case of front/back of a shirt pull them together into one image).

**Comment by raphaelm** [comment]
> Not really, this link would open in the same tab

Would it? That's a bug, it should open in an new tab.

> I'd see this as a low-prio feature request

Same. It sure is useful, but I'm not even 100% sure on the tradeoff of usefulness and interface complexity here :(
